Dfe.Data.SearchPrototype/Web/Tests/APITests/ApiSearchResults.cs

```diff
@@ -0,0 +1,101 @@
+﻿using Xunit;
+using Dfe.Data.SearchPrototype.Web.Tests;
+using Xunit.Abstractions;
+using Microsoft.AspNetCore.Mvc.Testing;
+using Newtonsoft.Json;
+using FluentAssertions;
+using Microsoft.AspNetCore.Http;
+using static Dfe.Data.SearchPrototype.Web.Tests.Shared.Helpers.ApiHelpers;
+
+namespace Dfe.Data.SearchPrototype.WebApi.Tests.APITests;
+
+public class ApiSearchResults : IClassFixture<PageWebApplicationFactory<Program>>
+{
+    private const string SEARCHKEYWORD_ENDPOINT = "/establishments?SearchKeyword=";
+    private readonly HttpClient _client;
+    private readonly ITestOutputHelper _logger;
+    private readonly PageWebApplicationFactory<Program> _factory;
+
+    public ApiSearchResults(PageWebApplicationFactory<Program> factory, ITestOutputHelper logger)
+    {
+        _client = factory.CreateClient(new WebApplicationFactoryClientOptions
+        {
+            AllowAutoRedirect = true
+        });
+        _factory = factory;
```

Comment:
I don't think we need this

Reply:
Future work but maybe we could look at injecting the HttpClient (pre-configured) rather than the PageWebApplicationFactory